microproxy/test/layer/test_socks.py

```diff
@@ -118,7 +112,7 @@ def test_socks_request_ipv4(self):
 
         self.assertIsNotNone(dest_stream)
         self.assertFalse(dest_stream.closed())
-        self.assertEqual(host, "127.0.0.1")
+        self.assertEqual(str(host), "127.0.0.1")
```

Comment:
I think ```self.assertEqual(host, u"127.0.0.1")``` would better if host type is unicode.
That it's more clearly to know what's the type of host.